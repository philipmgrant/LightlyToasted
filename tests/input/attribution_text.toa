Testing1
Testing2
Testing3
|AttributionText: via test process