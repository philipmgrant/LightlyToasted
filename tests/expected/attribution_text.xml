<toast><visual><binding template="ToastImageAndText04"><image id="1" src=""/><text id="1">Testing1</text><text id="2">Testing2</text><text id="3">Testing3</text><text placement="attribution">via test process</text></binding></visual></toast>